.appveyor.yml

```diff
@@ -36,6 +37,12 @@ install:
           $env:HOST="arm-w32-cygwin"
           $env:MSVCC="/cygdrive/c/projects/libffi/msvcc.sh -marm"
           $env:SRC_ARCHITECTURE="arm"
+        } ElseIf ($env:Platform -Match "arm64") {
```

Comment:
For "arm64", Appveyor matches "arm" first, and makes it to use the ARM32 toolchain instead of the ARM64 toolchain.
You can see this behavior in https://ci.appveyor.com/project/atgreen/libffi/builds/24605359/job/pagcp8q8htpxynna?fullLog=true.
You might want to consider to move arm64 above of arm.

Reply:
Thanks!  Good catch.  I have submitted a fix.